src/category_theory/sites/sheaf.lean

```diff
@@ -156,6 +156,18 @@ def Sheaf_equiv_SheafOfTypes : Sheaf J (Type w) ≌ SheafOfTypes J :=
 instance : inhabited (Sheaf (⊥ : grothendieck_topology C) (Type w)) :=
 ⟨(Sheaf_equiv_SheafOfTypes _).inverse.obj (default _)⟩
 
+variables {J} {A}
+
+/-- If the empty sieve is a cover of `X`, then `F(X)` is terminal. -/
+def Sheaf.is_terminal_of_bot_cover (F : Sheaf J A) (X : C) (H : ⊥ ∈ J X) :
```

Comment:
If `C` has an initial object, I guess that is always covered by the empty sieve, right? Could you please add that special case?

Reply:
I don't think this is the case. The empty sieve is never a cover in the trivial topology.